frontend/src/views/producer/dashboard/edition-form/CoverImage.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import { InputField } from '../../../../components/form';
+import Button from '../../../../components/button';
+
+const Image = styled.img`
+  width: 100%;
+  height: auto;
+  margin-bottom: 2rem;
+  border-radius: 8px;
+`;
+const FlexBox = styled.div`
+  display: flex;
+  width: 100%;
+  align-items: center;
+`;
+
+class CoverImage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      confirmedSrc: props.value,
+    };
+  }
+  onConfirm = () => {
+    this.setState({ confirmedSrc: this.props.value });
+  };
+  render() {
+    const { inputBackgroundColor, value, onChange } = this.props;
+    return (
+      <div>
+        <Image src={this.state.confirmedSrc} />
+        <FlexBox>
+          <InputField
+            style={{ width: '100%', marginRight: '1rem' }}
+            label="Image URL"
+            lightMode
+            backgroundColor={inputBackgroundColor}
+            type="text"
+            onChange={this.props.onChange}
```

Comment:
You are taking the onChange from the props already on line 29, so here you could just say: `onChange={onChange}` instead of this.props.onChange... and same goes for the value at next line

Reply:
this is substaintially changed in my next work (sorry for really late response)